.github/workflows/test.yml

```diff
@@ -22,6 +22,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: 1.82.0
```

Comment:
NOTE: Fix for lint error
To upgrade to 1.83, we need to fix some lint warnings.

Reply:
Fixed at https://github.com/datachainlab/ethereum-light-client-rs/pull/23/commits/021ca433304dbdfcc26248680c791e38820992cc